automation/terraform/infrastructure/us-east1.tf

```diff
@@ -222,21 +222,21 @@ resource "kubernetes_storage_class" "east1_standard" {
 ## Monitoring 
 
 provider helm {
-  alias = "helm_east1"
+  alias = "helm_east"
```

Comment:
Legacy setting based on existing cluster terraform state file. Should replace w/ `east1` once it's ok to completely wipe and refresh the state.